config/initializers/_load_config.rb

```diff
@@ -96,6 +91,6 @@
   default.reverse_merge!(options_from_config)
 end
 
-if Rails.env.production?
-  Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
-end
+Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
```

Comment:
I think this 3 rails config should not be part of the errbit config.
They are rails standards, and they are mostly hard-coded on the right environment file.
Just `secret_key_base` under production is one that uses a ENV var, but I would say we should still use it like that.